### PR TITLE
BAEL-6334 Add example code for checking if a list element is in anoth…

### PR DESCRIPTION
**Pull request by sam-gardner**
…er list

### CODE REVIEW

File: core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import com.baeldung.list.Country;
+
+public class ListContainsElementFromOtherListTest {
+
+    final private List<String> listOfLetters = Arrays.asList("a", "b", "c", "d");
+    final private List<String> listOfLettersWithOverlap = Arrays.asList("d", "e", "f", "g");
+    final private List<String> listOfCities = Arrays.asList("London", "Berlin", "Paris", "Brussels");
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingCollectionsDisjoint_thenDetectElementsInTwoLists() {
```
Comment (vinipx) [inline core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java:23]: let's remove the **Whole** word

File: core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        boolean shouldBeTrue = !Collections.disjoint(listOfLetters, listOfLettersWithOverlap);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = !Collections.disjoint(listOfLetters, listOfCities);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingStreams_thenDetectElementsInTwoLists() {
```
Comment (vinipx) [inline core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java:32]: let's remove the **Whole** word

File: core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          .anyMatch(listOfLettersWithOverlap::contains);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = listOfLetters.stream()
+          .anyMatch(listOfCities::contains);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingApacheCollectionUtils_thenDetectElementsInTwoLists() {
```
Comment (vinipx) [inline core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java:43]: let's remove the **Whole** word

File: core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        boolean shouldBeTrue = CollectionUtils.containsAny(listOfLetters, listOfLettersWithOverlap);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = CollectionUtils.containsAny(listOfLetters, listOfCities);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholePorpertiesInObjectsToCompare_whenUsingStreams_thenDetectElementsInTwoLists() {
```
Comment (vinipx) [inline core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java:52]: let's remove the **Whole** word
also please fix the typo with **Porperties** -> **Properties**

**Review by vinipx — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java (hunk 4 above)
Comment (sam-gardner) [reply core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java]: I've taken **Whole** out of all of the test names and fixed the typo

**Review by vinipx — APPROVED** [review]

**Pull request merged**